packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/triggers/MonthlyTrigger.java

```diff
@@ -0,0 +1,85 @@
+package expo.modules.notifications.notifications.triggers;
```

Comment:
shall we write new code in Kotlin instead?

Reply:
ok, let's do that here, though the interop between Kotlin and Java is pretty seamless so I'd advocate for writing future code in Kotlin only.